recipes/vamos/meta.yaml

```diff
@@ -1,19 +1,26 @@
-{% set version = "1.3.6" %}
+{% set version = "2.1.0" %}
 {% set name = "vamos" %}
 
 package:
   name: vamos
   version: {{ version }}
 
 source:
-  url: https://github.com/ChaissonLab/vamos/archive/refs/tags/{{ version }}.tar.gz
-  sha256: f5379fbb35bd664b761a08314d7cddc6d0d7ac3faa4c02273083626942755b10
+  url: https://github.com/ChaissonLab/vamos/archive/refs/tags/vamos-v{{ version }}.tar.gz
+  sha256: 3071eeb263cd0d9cdbe74b04ac5ef6d8e6d46d0d871295d0ccb0cab7762b7d77
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x') }}
+
+host:
+  - linux
```

Comment:
I have never seen such `host` property before.
The right way to skip the build on osx is `skip: True  # [osx]` that you actually removed below. But the build on osx is successful: https://github.com/bioconda/bioconda-recipes/pull/44840/checks?check_run_id=26159072055